src/Catty UITests/ScriptCollectionView/BrickCellTests.swift

```diff
@@ -25,6 +25,8 @@ import XCTest
 class BrickCellTests: XCTestCase {
 
     var app: XCUIApplication!
+    let testLookNames_toType: [String] = ["_test1", "_test2", "_test3"]
```

Comment:
Your test looks very nice, thanks! What is the difference between `testLookNames_toType` and `testLookNames`? I can not figure it out when reading the variable names. Do we need both?

Reply:
Yes, we will need both. 
When a new look is created and saved, an alert appears with a text field to save it with a name. By default that text field already have a text "look". So previously, I had used backspace character and removed it and then used values from _testLookNames_. But it made the code look more complicated for no reason (Because, Swift don't have \b that's why we will need another variable which can store unicode equivalent for it). So then I decided to append __test#_  from _testLookNames_toType_ at the end and then when adjusting picker, we will need _testLookNames_.

Reply:
I see! Would maybe `alert3.textFields.buttons["Clear text"].tap()` be a potential solution for this? One minor suggestion: could you please reorder your private methods to the top (below `tearDown`) so that all actual test functions are next to each other?

Reply:
Thank you so much! :-) 
I completely missed the existence of that clear button.